docs/assets/css/main.css

```diff
@@ -15556,9 +15556,6 @@ a.text-dark:focus {
     .td-content>.lead,
     .td-content>h1,
     .td-content>h2,
-    .td-content>ul,
```

Comment:
Maybe... adding this back would do the trick for restoring right sidebar?

Reply:
No, I tried it didn't work. @alphaX86